marlowe-runtime-cli/app/Language/Marlowe/Runtime/CLI/Command/Create.hs

```diff
@@ -1,3 +1,5 @@
+-- editorconfig-checker-disable-file
+
```

Comment:
Can remove

Reply:
Fixed in 34f1d7e5d89e68996254f165386a3264372c3252.